libs/SalesforceSDK/src/com/salesforce/androidsdk/rest/ClientManager.java

```diff
@@ -305,92 +301,74 @@ public void removeAccounts(Account[] accounts) {
     }
 
     /**
-     * Create a new account and return the details of the new account in a bundle.
-     * @param accountName
-     * @param username
-     * @param refreshToken
-     * @param authToken
-     * @param instanceUrl
-     * @param loginUrl
-     * @param idUrl
-     * @param clientId
-     * @param orgId
-     * @param userId
-     * @param passcodeHash
-     * @return
+     * Creates a new account and returns the parameters as a Bundle.
+     *
+     * @param accountName Account name
+     * @param username Username.
+     * @param refreshToken Refresh token.
+     * @param authToken Access token.
+     * @param instanceUrl Instance URL.
+     * @param loginUrl Login URL.
+     * @param idUrl Identity URL.
+     * @param clientId Client ID.
+     * @param orgId Org ID.
+     * @param userId User ID.
+     * @param clientSecret Client secret.
+     * @param communityId Community ID.
+     * @param communityUrl Community URL.
+     * @param firstName First name.
+     * @param lastName Last name.
+     * @param displayName Display name.
+     * @param email Email.
+     * @param photoUrl Photo URL.
+     * @param thumbnailUrl Thumbnail URL.
+     * @param additionalOauthValues Additional OAuth values.
+     * @return Account info.
      */
     public Bundle createNewAccount(String accountName, String username, String refreshToken,
     		String authToken, String instanceUrl, String loginUrl, String idUrl,
-    		String clientId, String orgId, String userId, String passcodeHash) {
-        return createNewAccount(accountName, username, refreshToken, authToken,
-                                instanceUrl, loginUrl, idUrl, clientId, orgId,
-                                userId, passcodeHash, null);
-    }
-
-    public Bundle createNewAccount(String accountName, String username, String refreshToken,
```

Comment:
Took the opportunity to clean up and consolidate these methods. We've been carrying some tech debt in this class for a while now.